demos/ASP.Net Core/ITOps.ViewModelComposition/ServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,38 @@
+﻿using Microsoft.Extensions.DependencyInjection;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+
+namespace ITOps.ViewModelComposition
+{
+    public static class ServiceCollectionExtensions
+    {
+        public static void AddViewModelComposition(this IServiceCollection services, string assemblySearchPattern = "*ViewModelComposition*.dll")
+        {
+            var fileNames = Directory.GetFiles(AppContext.BaseDirectory, assemblySearchPattern);
+
+            var types = new List<Type>();
+            foreach (var fileName in fileNames)
+            {
+                var temp = AssemblyLoader.Load(fileName)
+                    .GetTypes()
+                    .Where(t =>
+                    {
+                        var typeInfo = t.GetTypeInfo();
+                        return !typeInfo.IsInterface 
```

Comment:
it's a `!`, we're looking for types that are neither interfaces or abstract classes. How would you change it to make sure we'll pick only concrete types?